client/src/store/auth/auth.selectors.ts

```diff
@@ -9,6 +9,11 @@ export const selectUserSession = createSelector(
     (authState: AuthState) => authState?.userSession
 );
 
+export const selectEmail = createSelector(
```

Comment:
As we discussed, in order to avoid mistakes in the future, `auth` state should save only `accessToken` / `refreshToken` (update backend too if necessary), and in `userDetails` save `email, gender, age, etc`.
There are multiple places that use `auth` selectors, change them accordingly and make sure we don't insert other bugs by mistake because it's quite sensitive

Reply:
done